core/nodejs8Action/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'eclipse'
+eclipse {
+    project {
+        natures 'org.eclipse.wst.jsdt.core.jsNature'
+        buildCommand 'org.eclipse.wst.jsdt.core.javascriptValidator'
+    }
+}
+
+ext.dockerImageName = 'nodejs8action'
+apply from: '../../gradle/docker.gradle'
+distDocker.dependsOn 'copyProxy'
+distDocker.finalizedBy('rmProxy')
+
+task copyProxy(type: Copy) {
+    from '../nodejsActionBase'
```

Comment:
@markusthoemmes need some help here:
made changes to 
```
ext.dockerImageName = 'nodejs8action'
apply from: '../../gradle/docker.gradle'
distDocker.dependsOn 'copyFiles'
distDocker.finalizedBy 'rmFiles'

def runners = files(
    new File(project(':core:nodejsActionBase').projectDir)
)

task copyFiles(type: Copy) {
    from runners
    into '.'
}

task rmFiles(type: Delete) {
    delete runners.collect { it.getName() }
}
```
But I get errors running `distDocker`
```
* What went wrong:
A problem occurred evaluating project ':core:nodejs8Action'.
> Could not find matching constructor for: java.io.File(java.io.File)

```